roles/openshift_sdn/files/sdn.yaml

```diff
@@ -77,16 +77,24 @@ spec:
             fi
           done
 
+          #BUG 1728081: If /etc/sysconfig/origin-node got mounted as DirectoryOrCreate; clean it up so we can ultimately mount /etc/sysconfig/origin-node as FileOrCreate
+          if [[ -d /etc/sysconfig/origin-node ]]; then
+            echo "Deleting the origin-node dir"
+            rmdir /etc/sysconfig/origin-node
+          fi
+          
           # Take over network functions on the node
           rm -Rf /etc/cni/net.d/80-openshift-network.conf
           cp -Rf /opt/cni/bin/* /host/opt/cni/bin/
 
           # Load DEBUG_LOGLEVEL
           if [[ -f /etc/sysconfig/origin-node ]]; then
             set -o allexport
+            echo "Reading the options from origin-node directory"
```

Comment:
"Reading environment variables from /etc/sysconfig/origin-node" (and similar below)